Comp2024/src/main/java/frc/robot/subsystems/Feeder.java

```diff
@@ -0,0 +1,51 @@
+//
+// Intake Subystem - takes in Notes and delivers them to the Shooter
+//
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+//
+// Intake subsystem class
+//
+public class Feeder extends SubsystemBase
+{
+  // Member objects
+  private final WPI_TalonSRX m_feeder       = new WPI_TalonSRX(0);
```

Comment:
My estimate is:
- feeder roller
- feeder rotary
- MAYBE a CANCoder
- Limit switch for detecting a note acquired
- Limit switch for zeroing rotary mechanism IF no CANCoder